### PR TITLE
Change IRC link in community.md

### PR DESCRIPTION
**Pull request by blutack**
The IRC link for #elixir-lang has been updated, this updates the phoenix docs to match.

### CODE REVIEW

File: guides/introduction/community.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The Elixir and Phoenix communities are friendly and welcoming. All questions and
 
 There are a number of places to connect with community members at all experience levels.
 
-  * We're on Freenode IRC in the [\#elixir-lang](http://webchat.freenode.net/?channels=elixir-lang) channel.
+  * We're on Libera IRC in the [\#elixir-lang](https://web.libera.chat) channel.
```
Comment (mcrumm) [inline guides/introduction/community.md:7]:
```suggestion
  * We're on Libera IRC in the [\#elixir](https://web.libera.chat/?channels=#elixir) channel.
```

**Review by mcrumm — COMMENTED** [review]
Thanks @blutack! I made a slight update to align this link with the IRC link in the README :)

**Comment by blutack** [comment]
Cheers @mcrumm - that channel preselection param doesn't seem to work with Libera's web client so I removed it to avoid confusion. Should have said in the original pull request!

**Comment by mcrumm** [comment]
🙈 would you like to send one more PR to fix both URLs?

**Comment by blutack** [comment]
Ahh nope I didn't realise you'd added the # prefix on - that fixes it for Libera!

**Comment by mcrumm** [comment]
Sweet, thanks for the update!

**Pull request merged**